Exercises.py

```diff
@@ -92,11 +91,9 @@
 
 
 # 10. Create a string of length 10 with empty spaces in it. change empty spaces with '+' sign
-
 str_new = ' ' * 10
-str_new = '+' * 10
+str_new = str_new.replace('', '+') # string method to replace empty spaces with '+'
```

Comment:
check the result
yo did not replaced empty spaces

Reply:
fixed